File: TakeFive/takeFive/src/main/java/com/teuskim/takefive/common/Cell.java
package com.teuskim.takefive.common;

import android.graphics.Point;

public class Cell {

	private int x, y;
	private boolean isUserTurn;
	
	public Cell(Point point, boolean isUserTurn) {
		this.x = point.x;
		this.y = point.y;
		this.isUserTurn = isUserTurn;
	}
	
	public int getX() {
		return x;
	}
	
	public void setX(int x) {
		this.x = x;
	}

	public int getY() {
		return y;
	}
	
	public void setY(int y) {
		this.y = y;
	}
	
	public boolean isUserTurn() {
		return isUserTurn;
	}

}

File: TakeFive_kotlin/takeFive/src/main/java/com/teuskim/takefive/common/Cell.kt
package com.teuskim.takefive.common

import android.graphics.Point

class Cell(point: Point?, val isUserTurn: Boolean) {

    var x: Int = 0
    var y: Int = 0

    init {
        if (point != null) {
            this.x = point.x
            this.y = point.y
        }
    }

}
